### PR TITLE
Booking now creates a booking in db

### PR DESCRIPTION
**Pull request by adrianchen91**
Booking now creates a book. 
Very basic check atm. - Checks for existing entry for the car only at this point.

More comprehensive check to be implemented next sprint. 

Management console front end is complete. 

Clean up burger menu. - Profile only displayed if logged in.
Manage only displayed if admin

### CODE REVIEW

File: client/modules/Booking/Booking.js [hunk 1]
```diff
@@ -117,19 +120,45 @@ labels = {
...
+				this.setState({booked: true});
+			})
+			.catch(err => console.log(err));
+		}
+		//alert(`Signed up with email: ${email} password: ${password1} mobile: ${mobile} license: ${license}`);
+}
+
 	handleSubmit(evt){
        if (this.isFormInvalid()) {
          evt.preventDefault();
```
Comment (s3337852) [inline client/modules/Booking/Booking.js]: can we move evt.preventDefault to the first line of this method? currently we are calling it as the first step of both the if and the else statements

Comment (adrianchen91) [reply client/modules/Booking/Booking.js]: Gimme a sec, will address as soon as I get my mac running

File: server/controllers/booking/booking.controller.js [hunk 2]
```diff
@@ -1,13 +1,39 @@
...
+  User.find({'_id': userid})
+  .exec((usrerr, selecteduser) => {
+    if (usrerr) {
+      res.status(500).send(usrerr);
+    } else if (!selecteduser) {
+      res.status(404).send();
+    } else {
+
+  car.find({'_id': carid})
+  .exec((carerr, vehicle) => {
```
Comment (s3337852) [inline server/controllers/booking/booking.controller.js]: can we use camel case for variables ie carErr instead of carerr

Comment (adrianchen91) [reply server/controllers/booking/booking.controller.js]: sure

File: client/modules/Cars/Cars.js [hunk 3]
```diff
@@ -129,13 +129,6 @@ export class Car extends Component {
             </form>
```
Comment (s3337852) [inline client/modules/Cars/Cars.js]: can we make sure that the indentation is consistent. there should be a code formatter tool built into vs code, i think it is ALT + SHIFT + F... either /form is out of place here or <Row is

**Pull request merged**